tools/packages/solaris/build.properties

```diff
@@ -1,120 +0,0 @@
-###############################################################################
```

Comment:
we should delete the solaris build altogether, I doubt it works, and it's definitely unmaintained